index.bs

```diff
@@ -1,8 +1,9 @@
 <pre class='metadata'>
 Title: Web Speech API
 Level:
-Status: CG-DRAFT
-Group: Audio Working Group
+Status: WD
+Group: audiowg
```

Comment:
I think we can leave these out from the PR.

Technically the Web Speech API is still managed by Audio CG, not WG.